ProcessSlave.php

```diff
@@ -332,46 +337,48 @@ public function commandBootstrap(array $data, Connection $conn)
     /**
      * Handles incoming requests and transforms a $request into a $response by reference.
      *
-     * @param \React\Http\Request $request
-     * @param HttpResponse        $response
+     * @param RequestInterface $request
      *
+     * @return ResponseInterface
      * @throws \Exception
      */
-    public function onRequest(\React\Http\Request $request, HttpResponse $response)
+    public function onRequest(RequestInterface $request)
     {
         $this->prepareEnvironment($request);
 
+        $response = $this->handleRequest($request);
```

Comment:
Catch will be better than propagating exception further, but maybe only logging will be sufficient? With bad implementation of Bootstrap, Bridge we can end up here with app exceptions. IMO sometimes it's not a reason to close slave. Logging and sending 500 - Internal Server Error should be OK. What do You think?